app/src/interfaces/file/file.vue

```diff
@@ -89,6 +102,24 @@ const edits = computed(() => {
 	return props.value;
 });
 
+const values = inject('values', ref<Record<string, any>>({}));
+
+const customFilter = computed(() => {
+	return parseFilter(
+		deepMap(props.filter, (val: any) => {
```

Comment:
```suggestion
const values = inject('values', ref<Record<string, unknown>>({}));

const customFilter = computed(() => {
	return parseFilter(
		deepMap(props.filter, (val: unknown) => {
```

`unknown` would be a safer type choice here than `any` I think, `any` has the tendency to hide potential type issues.